integrations/k8s-using-deployment/falco-event-generator-job.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: falco-event-generator
+spec:
+  template:
+    spec:
+      containers:
+      - name: event-generator
+        image: falcosecurity/event-generator
+        command: ["./event-generator", "run"]
```

```suggestion
        command: ["./event-generator", "run", "syscall"]
```
Same as above without `--loop`